### PR TITLE
feat: display Parent NFT data on the Parent page

### PR DESCRIPTION
**Pull request by impelcrypto**
* feat: display Parent NFT date on Parent page
http://localhost:8080/#/shibuya-testnet/parent?tokenId=1

image

image

### CODE REVIEW

**Review by bobo-k2 — APPROVED** [review]

**Comment by impelcrypto** [comment]
@bobo-k2 
FYI: I've added chunky and parts contract addresses in [providerEndpoints](https://github.com/AstarNetwork/ink-redible-nft/blob/1b25ec0c14ab1e2c076c7ab4096acefbe5de3ded/src/config/chainEndpoints.ts#L104)

File: src/components/parent/Inventory.vue [hunk 1]
```diff
@@ -75,13 +108,27 @@ export default defineComponent({
 
     const dummyEquippedList = [dummyItemB, dummyItemA];
 
+    const equipped = computed<IBasePart[]>(() => props.parts.filter((it) => isSlotEquipped(it)));
+
+    const navigateToChild = (id: string): void => {
```
Comment (impelcrypto) [inline src/components/parent/Inventory.vue:113]: Hi @bobo-k2 
For the equipped items, I assume `tokenId` for the "/child page" can be found in `z`. Please let me know if this is not correct.
image

Comment (impelcrypto) [reply src/components/parent/Inventory.vue:113]: image

Comment (bobo-k2) [reply src/components/parent/Inventory.vue:113]: Hi @impelcrypto this is not correct. Z variable determines in which layer asset should be rendered. Something like z-index in css.
To get token id yo ushould use `equippable::get_equipment`. The method, beside others, returns equipped item contract address and token id.

**Comment by bobo-k2** [comment]
> @bobo-k2 FYI: I've added chunky and parts contract addresses in [providerEndpoints](https://github.com/AstarNetwork/ink-redible-nft/blob/1b25ec0c14ab1e2c076c7ab4096acefbe5de3ded/src/config/chainEndpoints.ts#L104)

Maybe to make naming more generic. Instead of `chunkyAddress` name it `baseContractAddress` and make it array, so we can support multiple NFT contracts. `partsAddress` is not needed since this address can be fetched from base contract.

**Comment by impelcrypto** [comment]
> partsAddress is not needed since this address can be fetched from base contract.
Sorry, could you send me an example code for this?

**Comment by bobo-k2** [comment]
> > partsAddress is not needed since this address can be fetched from base contract.
> > Sorry, could you send me an example code for this?

Sure,
If you want to get assets that can be equipped to base you can call
image
As result you will get child contract address and token id.

When you want to check what is equipped on base you can call
image

**Comment by impelcrypto** [comment]
@bobo-k2 

> Maybe to make naming more generic. Instead of chunkyAddress name it baseContractAddress and make it array,

Fixed in [this](https://github.com/AstarNetwork/ink-redible-nft/pull/10/commits/fc2e9cf2493325a795b1f4b8cbc759a2c377f7c3#diff-bbb15aefae956e4e9ba6a9527a96e3f2693a2d247e6c2235d13d8f25f3274259R22) commit.

>  partsAddress is not needed since this address can be fetched from base contract.

We still need this value for now for [this](https://github.com/AstarNetwork/ink-redible-nft/blob/fc2e9cf2493325a795b1f4b8cbc759a2c377f7c3/src/modules/nft/read-token/index.ts#L107) line. Let's take a look again once [this](https://github.com/rmrk-team/rmrk-ink/issues/46) issue has been resolved.

**Pull request merged**